### PR TITLE
text changes

### PR DESCRIPTION
**Pull request by david-piiano**
And also renamed the CI folder to CI

### CODE REVIEW

File: ci/README.md [hunk 1]
```diff
@@ -40,8 +43,7 @@ on:
       sub_dir:
         required: false
         type: string
-        default: ''
-
+        default: ""
```
Comment (david-piiano) [inline ci/README.md:46]: I don't know if this matters - it is automatic vscode refactoring

File: ci/README.md [hunk 2]
```diff
@@ -3,27 +3,30 @@
...
-Client Id
-Client Secret (These tokens are described [here](../cli/) )
-A URL to the repository (with any authentication token needed to clone it)
-The sub-directory to scan (optional)
-Customer Identifier: <your-company-name>
-Customer Environment: <environment-such-as-prod-or-stage>
-Notes:
+1. Client Id ([more details](../cli/))
+2. Client Secret ([more details](../cli/))
+3. A URL to the repository (with any authentication token needed to clone it)
```
Comment (nerya10) [inline ci/README.md:20]: Do we need to clone it ?
We are running on local folder

File: ci/README.md [hunk 3]
```diff
@@ -3,27 +3,30 @@
...
+4. The sub-directory to scan (optional)
+5. Customer Identifier: <your-company-name>
+6. Customer Environment: <environment-such-as-prod-or-stage>
 
-The provided runner_type should have netcat installed.
+**Prerequisites:** the provided runner_type should have netcat installed.
```
Comment (nerya10) [inline ci/README.md:25]: Maybe it worth to mention why do we need that ?

File: ci/README.md (hunk 2 above)
Comment (nerya10) [reply ci/README.md:20]: In the GH action it written as :

3. A directory to scan

**Review by nerya10 — APPROVED** [review]

**Pull request merged**